shell/platform/windows/flutter_windows_view_unittests.cc

```diff
@@ -132,6 +132,110 @@ class MockAngleSurfaceManager : public AngleSurfaceManager {
 
 }  // namespace
 
+// Ensure that submenu buttons have their expanded/collapsed status set
+// apropriately.
+TEST(FlutterWindowsViewTest, SubmenuNativeState) {
```

Comment:
Optional: `SubMenuExpandedState` to be slightly more precise?

Reply:
Thanks a lot for helping review! Just updated:)